easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author: Adam Huffman
+# adam.huffman@crick.ac.uk
+# The Francis Crick Institute
+# Based on work by npklein
+
+easyblock = 'PackedBinary'
+
+name = 'sratoolkit'
+version = '2.5.7'
+
+homepage = 'http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software'
+description = """The SRA Toolkit, and the source-code SRA System Development 
+ Kit (SDK), will allow you to programmatically access data housed within SRA
+ and convert it from the SRA format"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/%(version)s/']
+sources = ['sratoolkit.%(version)s-centos_linux64.tar.gz']
+
+executables = [
```

Comment:
Ah, I had assumed it was required and just re-used for convenience in the original PR.

Old RPM habits die hard, I suppose.